src/Form/StrawberryfieldToolsForm.php

```diff
@@ -0,0 +1,157 @@
+<?php
+
+namespace Drupal\strawberryfield\Form;
+
+use Drupal\Core\DependencyInjection\ContainerInjectionInterface;
+use Drupal\Core\Entity\EntityRepositoryInterface;
+use Drupal\Core\Form\FormBase;
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\Core\Render\RendererInterface;
+use Drupal\node\NodeInterface;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+use Drupal\Core\Ajax\AjaxResponse;
+use Drupal\strawberryfield\Ajax\UpdateCodeMirrorCommand;
+
+/**
+ * Returns responses for Node routes.
+ */
+class StrawberryfieldToolsForm extends FormBase {
+
+  /**
```

Comment:
Line 20 to line 57 can one removed. This is form is only using what FormBase is already providing. Please test (clearing caches bc Forms are cached)

Reply:
Resolved by 99a8e21bf3275f9956960135b93aef4c5df5d120

Reply:
@aksm hi, this is not resolved yet. The constructor is still using  it (and still present, we don't need it)

again, we can delete __construct and create here since there is nothing we need to override from the base form

Reply:
@DiegoPino Sorry, thought I had already removed those but missed them. Now resolved in 652bd8e774941c04c854c869a8ebaf33b644f67e